ui/raidboss/data/05-shb/trial/wol-ex.ts

```diff
@@ -377,7 +377,7 @@ const triggerSet: TriggerSet<Data> = {
       // This is still 1 second before this cast goes off, giving ~7 seconds before LB is needed.
       delaySeconds: 4,
       alarmText: (data, _matches, output) => {
-        if (data.role === 'tank')
+        if (data.role === 'tank' || data.job === 'BLU')
```

Comment:
Mmmm, maybe? What if blu doesn't get LB? This is fine for now I guess, but maybe should say something else if this isn't the way to handle this.

Reply:
They need some type of call at least, and I was going for minimum changes.  Currently, BLUs substitute Diamondback where you would use a tank LB, so they should get those calls.